libs/application/template-api-modules/src/lib/types/index.ts

```diff
@@ -21,6 +21,7 @@ export interface BaseTemplateAPIModuleConfig {
   }
   attachmentBucket: string
   presignBucket: string
+  bypassPayments: boolean
```

Comment:
Is this needed?

Reply:
Nope, missed that in my cleanup, thanks for the catch.